spec/features/attendances/create_slack_attendance_spec.rb

```diff
@@ -18,7 +18,7 @@
       stub_request(:get, "https://slack-attendance-service.herokuapp.com/api/v1/attendance?channel_id=#{@channel_id}&timestamp=#{@timestamp}") \
       .to_return(body: File.read('spec/fixtures/slack/message_replies_response.json'))
 
-      stub_request(:post, ENV['POPULI_API_URL']).
+      stub_request(:post, ENV['POPULI_API_URL'] || "https://fake-populi-domain.com").
```

Comment:
They wouldn't be able to pull the default from the populi service. We aren't stubbing a method in the populi service, we are stubbing an HTTP call, so this stub isn't directly connected to the populi service, just the http call it sends. 

I agree it would be nice to have a default value in 1 place. The issue is that since we're pulling this value from the environment variables, we would need to create that default at the environment level and I haven't found a good way to do that yet.